tests/licensedcode/data/datadriven/lic1/alexa-skills-kit-sdk-for-java.txt.yml

```diff
@@ -1,2 +1,4 @@
 license_expressions:
   - apache-2.0
+  - apache-2.0
+  - unknown-license-reference
```

Comment:
I'm suspecting that I've added a key phrase to a rule that prevents it from matching, but now also prevents `unknown-license-reference` for being removed as contained.

See which Apache rule it was and see if key phrase can be less strict, or a new rule must be added.

Reply:
`src/licensedcode/data/rules/apache-2.0_151.RULE` would match probably if only it didn't have the word `licensed`. Instead I've added a new rule `apache-2.0_1026.yml` for now.